Isolate the Django ORM test from App Engine.

dhermes opened the pull request:
Since most of our tests run with `--ignore-files=test_appengine\.py`, the App Engine SDK is "never" available when this test runs in our set-up.

nathanielmanistaatgoogle commented:
I don't understand this - tell me more about it?

dhermes commented:
The `tox -e cover` environ essentially does as follows

```
nosetests ...(LOTS OF FLAGS)... --ignore-files=test_appengine\.py
nosetests ...(LOTS OF FLAGS)... --cover-package=oauth2client.appengine --gae-application=tests/data ...
coverage report --show-missing --fail-under 80
```

We intentionally isolate the App Engine tests because importing the SDK **really** messes with imports. (We also have `tox -e py27` and `tox -e gae` separate).

So in the coverage report, the only time `test_django_orm.py` is getting executed is in the first run through `nosetests`. As a result, the import of the GAE SDK never occurs.

----

Doing a `git blame` on this code, the `try` block was added in 02b5f3f369d9dceaf197eac540ce5797d1a07250 (September 2012).

Going back to that original review (https://codereview.appspot.com/6490081) it doesn't seem that the App Engine tests needed to be isolated from the others at that time.

nathanielmanistaatgoogle commented:
Why leave a note behind rather than just remove the try/import/except entirely and cleanly?

dhermes commented:
That's fine with me, I was just trying to leave some evidence of what may occur.

Shall I proceed and just remove the note?

nathanielmanistaatgoogle commented:
Yes I think so.

dhermes committed:
Isolate the Django ORM test from App Engine.

Since most of our tests run with --ignore-files=test_appengine\.py,
the App Engine SDK is "never" available when this test runs in
our set-up.

dhermes commented:
Note removed. PTAL.
